Files/Scripts/Options/Ocarina of Time.psm1

```diff
@@ -650,9 +651,25 @@ function ByteOptions() {
     # SWORD TRAIL COLORS #
 
     if ($Redux.Colors.SetSwordTrail -ne $null) {
-        if (IsColor   $Redux.Colors.SetSwordTrail[0]   -Not)   { ChangeBytes -Offset "BEFF7C" -Values @($Redux.Colors.SetSwordTrail[0].Color.R, $Redux.Colors.SetSwordTrail[0].Color.G, $Redux.Colors.SetSwordTrail[0].Color.B) }
```

Comment:
Explain what BEFF80 and BEFF88 are needed for.

Reply:
BEFF80 is the starting segment of the base color trail with BEFF88 being the ending segment version of the base color. I paired them with their respective segments of start to make their effects happen consistently with them, but this too I can show as well.